CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## v0.18
+
+### Features
+
+1. Add '--backend' flag to `ilab model serve` command to allow for specifying the backend to use
+   when serving a model. This is useful when you have multiple backends installed and want to
+   specify which one to use. Currently, the only supported backend is `llama`.
```

Comment:
llama is well known as Meta's LLM, maybe use llama.cpp/llama-cpp/llama-cpp-python is better?

Reply:
Good point thanks, I'll rename it to "llama-cpp" instead.